content/blog/tina-cloud-and-nextjs-the-perfect-match.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Tina Cloud and Next.js: the perfect match'
+date: '2021-05-13T14:34:20+02:00'
+author: Logan Anderson & Frank Taillandier
+last_edited: '2021-05-13T14:33:22.066Z'
+---
+Next.js is **our default choice** when it comes to managing static content, and it works perfectly with our new headless CMS [Tina Cloud](/cloud/).
+
+We jumped on the Next.js bandwagon as soon as we realized this would be a perfect companion for Tina, especially because Next.js is a rather unopinionated React framework, that gives you great flexibility when it comes to working with data at build time or at run time.
+
+Tina Cloud is a Git-backed CMS, Next.js is just what we need to deliver fast static sites that you can edit visually in context.
+
+Let’s see how Next.js makes a Tina developers' job easier.
+
+## Next.js: More Than a Static Site Generator
+
+Next.js is a popular, open-source meta-framework and is quickly becoming[ the standard for new React projects](https://www.npmtrends.com/next-vs-gatsby-vs-nuxt). Its hybrid nature means that it can be used for both static site generation (SSG) _and_ server-side rendering (SSR).
```

Comment:
```suggestion
Next.js is a popular, open-source meta-framework and is quickly becoming[ the standard for new React projects](https://www.npmtrends.com/next-vs-gatsby). Its hybrid nature means that it can be used for both static site generation (SSG) _and_ server-side rendering (SSR).
```